frontend/components/vault/WithdrawDaiFromVat.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown">
+            After liquidating a vault, the DAI-based liquidation incentive will end up in your VAT account. One more
+            transaction is required to move that DAI to your wallet.
+        </TextBlock>
+        <div class="my-4">
+            <WalletAuthorizationCheckPanel
```

Comment:
Do we want to add a `WalletConnectionCheck` panel here too? Incase someone views it without having their wallet connected? (Super edge case, I know!)